src/content/getting-started/management-cluster/index.md

```diff
@@ -0,0 +1,119 @@
+
+![Clusters listed in the Web UI](happa-clusters.png)
+
+Click "List clusters via the Management API" on the bottom to see which command to run for logging into your cluster.
+
+**Run this command in your shell.** The command performs authentication, creates a kubectl context and prints a message similar to this:
+
+```text
+$ kubectl gs login "https://api.<management cluster domain>/"
+[...]
+A new kubectl context has been created named 'gs-wombat' and selected. To switch back to this context later, use either of these commands:
+  kubectl gs login wombat
+  kubectl config use-context gs-wombat
+```
+
+When logging in, you have to authenticate in your browser using the configured identity provider (for example by signing into an Active Directory or GitHub user account).
+
+## Step 3: View resources on the management cluster
+
+Let's run some commands and understand what's happening under the hood.
+
+The first command is `kubectl get nodes`. This will show us the nodes making up our management cluster.
+
+```bash
+$ kubectl get nodes
+NAME                                          STATUS   ROLES                  AGE   VERSION
+ip-10-0-5-103.eu-central-1.compute.internal   Ready    worker                 19h   v1.24.10
+ip-10-0-5-121.eu-central-1.compute.internal   Ready    control-plane,master   19h   v1.24.10
+ip-10-0-5-14.eu-central-1.compute.internal    Ready    control-plane,master   19h   v1.24.10
+ip-10-0-5-163.eu-central-1.compute.internal   Ready    control-plane,master   19h   v1.24.10
+ip-10-0-5-183.eu-central-1.compute.internal   Ready    worker                 19h   v1.24.10
+ip-10-0-5-5.eu-central-1.compute.internal     Ready    worker                 19h   v1.24.10
+ip-10-0-5-53.eu-central-1.compute.internal    Ready    worker                 19h   v1.24.10
+```
+
+There are two kinds of roles: `control-plane` and `worker`. Note that this still relates to the management cluster. None of your business applications should ever run on the management cluster's nodes, unless you want to run custom operators.
+
+The second command we suggest running is `kubectl get orgs`. This will show the organizations that are defined in your management cluster.
+
+```bash
+$ kubectl get orgs
+NAME                    AGE
+production              97d
+testing                 97d
+giantswarm              97d
+```
+
+Organizations are a way to isolate clusters, apps, etc. between different teams or environments. More information can be found in [Organizations]({{< relref "/platform-overview/organizations" >}}).
+
+Finally, run the `kubectl gs get clusters -A` command, which will show you all the clusters managed by your management cluster. The `-A` flag stands for _all namespaces_.
+
+```bash
+$ kubectl gs get clusters -A
+NAMESPACE          NAME    AGE    CONDITION   RELEASE   ORGANIZATION
+org-testing        rfjh2   84d    UPDATED     18.4.0    testing
+org-production     jn88t   91d    CREATED     18.0.1    production
```

Comment:
I think this output may be outdated. If I run this command, I also see the `SERVICE PRIORITY` and `DESCRIPTION` columns

```
k gs get clusters -A                                                                                                     [gs-gorilla:default]
NAMESPACE                   NAME      AGE    CONDITION   RELEASE   SERVICE PRIORITY   ORGANIZATION            DESCRIPTION
org-giantswarm-production   rfjh2     576d   UPDATED     19.0.0    n/a                giantswarm-production   Giant Swarm Operations Platform
org-giantswarm-production   zj88t     583d   UPDATED     18.3.0    n/a                giantswarm-production   Team Atlas cluster
org-team-tinkerers          cicddev   28d    UPDATED     19.0.0    lowest             team-tinkerers          Giant Swarm CI/CD Platform - Non-production & CICD development
org-team-tinkerers          ei9o7     142d   UPDATED     19.0.0    highest            team-tinkerers          Giant Swarm CI/CD Platform
```

Reply:
updated